sockshop/catalogue/cmd/cataloguesvc/main.go

```diff
@@ -43,16 +38,24 @@ func main() {
 	}
 
 	// Data domain.
-	socks, tags, err := readFile(*file)
+	// TODO pull user/password from env?
+	db, err := sql.Open("mysql", "catalogue_user:default_password@tcp(catalogue-db:3306)/"+*dbName)
```

Comment:
Typically best to take the entire DSN `user:pass@tcp(host:port)/dbName?parseTime=true&otherParams=here` as a flag.